doc/faq.rst

```diff
@@ -15,14 +15,12 @@ system or be installed in trusted locations.
 
 Can Shifter import Docker images with unicode filenames embedded?
 -----------------------------------------------------------------
-Yes.  If you are seeing errors similar to the following in the image worker log
+Yes.  If you are seeing errors similar to the following in the image gateway log
 then you'll need to include the provide sitecustomize.py in your python setup
-or just point the celery process's PYTHONPATH to include it::
+or just point the gateway's PYTHONPATH to include it::
 
     [2016-08-01 09:41:20,664: ERROR/MainProcess] Task shifter_imagegw.imageworker.dopull[XXXXXXX-omitted-XXXXXXX] raised unexpected: UnicodeDecodeError('ascii', '/path/is/omitted/some\xc3\xa9_unicode', 35, 36, 'ordinal not in range(128)')
     Traceback (most recent call last):
-      File "/usr/lib/python2.6/site-packages/celery/app/trace.py", line 240, in trace_task
-          R = retval = fun(*args, **kwargs)
       File "/usr/lib/python2.6/site-packages/celery/app/trace.py", line 438, in __protected_call__
```

Comment:
The stack frame above (with an explicit reference to Celery) was removed, but this one remains.
Is there a specific reason?